app/src/main/res/layout/fragment_pokemon.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.fragment_pokemon` appears to be unused
 Unused resources make applications larger and slow down builds.